### ISSUE TITLE
Cursed books won't stack

### ISSUE THREAD

**Issue by jayrock3000** [pl3xgaming/Purpur#918]
### Expected behavior

Combining two cursed books in a grindstone should make a stack of two books which a player can shift click to pull into their inventory.

### Observed/Actual behavior

Recipe flashes correctly for a split second, then cancels out and doesn't allow you to combine the books.

### Steps/models to reproduce

Place two curse of binding books into a grindstone

### Purpur version

This server is running Purpur version git-Purpur-1492 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 29b24bc on HEAD)
You are 69 version(s) behind
Download the new version at: https://purpurmc.org/downloads
Previous version: git-Pufferfish-17 (MC: 1.18.1)

### Agreements

- [ ] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Purpur.

**Comment by katekerllenevich** [comment]
Word of advice, please actually read the agreements you made, instead of wasting peoples time.

> You are 69 version(s) behind
> I am running the latest version of Purpur available from https://purpurmc.org/downloads.

Also, you haven't provided a timings link, which is quite essential for providing support.

Feel free to update, and I would advise coming into the discord (we're more active there anyways).

**Comment by BillyGalbreath** [comment]
I went ahead and looked into this, and it seems to be a bug in our upstream, Paper.

I was able to recreate the issue on Paper build 200

![image](https://user-images.githubusercontent.com/332527/154649942-2705abc4-da33-4106-a5fc-86f498f78d07.png)

But not on Spigot

![image](https://user-images.githubusercontent.com/332527/154649991-4ad0d046-30e8-424b-87ae-2dd1a868b257.png)

I see you also tried to report the bug to PaperMC/Paper#7484 but you didn't use the Paper software when reporting it to them. Sadly, they are notorious for premature closure, ignoring, and blame shifting of bugs if they smell even a hint of another software. If you want them to take the bug seriously you'll have to reproduce it on a Paper server for them without any plugins and a default configuration and environment.
